content/docs/querying/functions.md

```diff
@@ -85,6 +95,15 @@ rate(http_requests_total{job="api-server"}[5m])
 
 `rate` should only be used with counters.
 
+## `round()`
+
+`round(v instant-vector, places=0 scalar)` rounds the sample values of all
```

Comment:
Good point, and admittedly this was the interface I was initially naively expecting as well. Would you be up for a PR around that (since you seem to be into the math)?